docs/source/v1.10.md.inc

```diff
@@ -4,9 +4,9 @@
 
 [//]: # (- Whatever (#000 by @whoever))
 
-[//]: # (### :warning: Behavior changes)
+### :warning: Behavior changes
 
-[//]: # (- Whatever (#000 by @whoever))
+- Empty room matching is now done for all sessions (previously only for the first session) for each subject. (#976 by @drammock)
```

Comment:
I think this qualifies more for the bug fix section below. Behavior changes are more meant for "deprecation"-type things where the default option or meaning of a specific option changes. Even if this did change which room was matched to some file, if the match is more correct, that's bugfix-ey rather than what is meant here by "behavior change". (All or almost all bugfixes are "behavior changes" of some sort!)